<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Обработчик</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/build-MorzyankaTRS-Desktop_Qt_5_12_2_MinGW_32_bit-Debug/icon/system-monitor-network.png</normaloff>:/build-MorzyankaTRS-Desktop_Qt_5_12_2_MinGW_32_bit-Debug/icon/system-monitor-network.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="fileTxtLBL">
     <property name="text">
      <string>Выберите файл с сообщением</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="fileTxtLED"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="fileTxtPBN">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/papka.png</normaloff>:/papka.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="fileMesLBL">
     <property name="text">
      <string>Введите сообщение</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="fileTxtLED_2"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="fileTxtPBN_2">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/papka.png</normaloff>:/papka.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="fileTxtLBL_2">
     <property name="text">
      <string>Выберите куда сохранить файл с результатом</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="logPTE">
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QPushButton" name="processPBN">
     <property name="text">
      <string>Обработать</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/65af2a2808c7a147fb4f6ba50286b126--psd-photoshop.jpg</normaloff>:/65af2a2808c7a147fb4f6ba50286b126--psd-photoshop.jpg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="fileMesLED"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fileTxtPBN</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>fileSelectSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>824</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileTxtPBN_2</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>fileSelectSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>824</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fileSelectSlot()</slot>
 </slots>
</ui>
